ganga/GangaCore/test/Unit/Utility/TestUtilitiesExecutePrint.py

```diff
@@ -40,10 +40,10 @@ def test_execute_cwd():
 def test_execute_output():
     ''' This tests the various levels when an import can occur and that complex objects can be returned via stdout '''
 
-    # Test printout of pickle dump interpreted correctly
-    d = execute('import pickle, datetime\nprint(pickle.dumps(datetime.datetime(2013,12,12)))', shell=False)
-    assert hasattr(d, 'month')
-    assert d.month == 12
+    # Test printout of pickle dump interpreted correctly - we have to use time here due to issues with datetime and pickling between 2 and 3
```

Comment:
The travis tests only use python3, so the subprocesses run with python 3. This therefore returns a string with a byte representation inside, i.e.:
```python
stdout = "b'\x80\x03ctime\nstruct_time\nq\x00(M\xe3\x07K\nK\x07K\x0bK\x0fK:K\x00M\x18\x01K\x01tq\x01}q\x02(X\x07\x00\x00\x00tm_zoneq\x03X\x03\x00\x00\x00BSTq\x04X\t\x00\x00\x00tm_gmtoffq\x05M\x10\x0eu\x86q\x06Rq\x07.'"
```

Reply:
for the datetime I think you can change the encoding to "latin"